package.json

```diff
@@ -307,6 +320,10 @@
             "default": null,
             "description": "Experimental: Enable modules auto-import following applying Tabnine suggestions"
           },
+          "tabnine.cloudHost": {
+            "type": "string",
+            "description": "The tabnine cloud host to connect to"
```

Comment:
Can we expose that only if on prem?

Reply:
yes it can be possible to add in the prep phase

Reply:
any idea how to append a nested object to this json from a cli call? (yarn,npm etc.) (without installing some fancy tool I suppose)

Reply:
I suppose this might work. I'll try 
https://www.npmjs.com/package/transform-json-webpack-plugin